cmd/context/create.go

```diff
@@ -52,6 +54,21 @@ func NewContextCommand() *ContextCommand {
 		LoginController:      login.NewLoginController(),
 		OktetoClientProvider: okteto.NewOktetoClientProvider(),
 		OktetoContextWriter:  okteto.NewContextConfigWriter(),
+		IsOktetoKubeTokenPresent: func(oktetoURL string) (bool, error) {
+			// this is a hack to check if the server was upgraded to a version that supports the /auth/kubetoken/{namespace} endpoint
+			// if the endpoint is not present, the server will return a 401 unauthorized error
+			kubeTokenURL, err := okteto.ParseOktetoURLWithPath(oktetoURL, "auth/kubetoken/default")
+			if err != nil {
+				return false, fmt.Errorf("failed to parse kubetoken url: %w", err)
+			}
+
+			resp, err := http.Get(kubeTokenURL)
+			if err != nil {
+				return false, fmt.Errorf("failed to get kubetoken url: %w", err)
+			}
+
+			return resp.StatusCode == http.StatusUnauthorized, nil
```

Comment:
```suggestion
			return resp.StatusCode != http.StatusUnauthorized, nil
```
if endpoint is not available backend response will be `401` but then we return `true` I think. We should return `true` when response is not `401` following func name

Reply:
@AdrianPedriza I got your confusion, but it's the other way around

> if endpoint is not available backend response will be 401

In fact, it's the opposite. if endpoint IS AVAILABLE backend response will be 401, since it can handle the request but we didn't add the auth mechanism. You can check this out by going to https://cloud.okteto.com/auth/kubetoken/default

If the endpoint is NOT AVAILABLE we'd get a 404

Reply:
cool then! 🚀